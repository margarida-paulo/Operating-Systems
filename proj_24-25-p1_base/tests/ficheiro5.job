SHOWSHOW
SHOW
SHOW
SHOW
SHOW
SHOW
SHOW
SHOW
