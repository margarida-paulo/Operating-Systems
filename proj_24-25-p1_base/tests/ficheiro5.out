Invalid command. See HELP for usage
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Leo, Tolstoy)
(Luís, Camões)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Oscar, Wilde)
(Pablo, Neruda)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Leo, Tolstoy)
(Luís, Camões)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Oscar, Wilde)
(Pablo, Neruda)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Leo, Tolstoy)
(Luís, Camões)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Oscar, Wilde)
(Pablo, Neruda)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Leo, Tolstoy)
(Luís, Camões)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Oscar, Wilde)
(Pablo, Neruda)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Cristina, Sousa)
(Carlos, Vinícius)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Emma, Stone)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Camões)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Nicolas, Mendes)
(Oscar, Wilde)
(Paulo, Carvalho)
(Pablo, Neruda)
(Renata, Teixeira)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Cristina, Sousa)
(Carlos, Vinícius)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Emma, Stone)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Camões)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Nicolas, Mendes)
(Oscar, Wilde)
(Paulo, Carvalho)
(Pablo, Neruda)
(Renata, Teixeira)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Vinícius)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Denzel, Washington)
(Emma, Stone)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Kate, Winslet)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Mari)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Robert, Downey)
(Ricardo, Oliveira)
(Renata, Teixeira)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tom, Hanks)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
